src/components/general/OutcomeCard.tsx

```diff
@@ -186,8 +170,27 @@ const OutcomeCard = (props: OutcomeCardProps) => {
   // If there is no outcome  value for active year, do not display card set
   if (goalOutcomeValue === undefined) return null;
 
+  const handleClickTab = () => handleClick(node.id);
+
+  const handleKeyDownOnTab = (e: React.KeyboardEvent) => {
```

Comment:
Interesting. Is this something we should also consider elsewhere?

Reply:
This is needed wherever we use a `div` as a tab, typically they're buttons (which would support hitting `Enter` out of the box), but since we have non-text content within tabs buttons don't work semantically here.